docs/Configuration/yaml-files.md

```diff
@@ -298,7 +352,7 @@ Fleet supports adding [GitHub environment variables](https://docs.github.com/en/
 - `$FLEET_VAR_NDES_SCEP_PROXY_URL`
 - `$FLEET_VAR_HOST_END_USER_EMAIL_IDP`
 
-Use `labels_include_all` to target hosts that have all labels in the array, `labels_include_any` to target hosts that have any label in the array, or `labels_exclude_any` to target hosts that don't have any of the labels in the array. Only one of `labels_include_all`, `labels_include_any`, or `labels_exclude_any` can be specified. If none are specified, all hosts are targeted.
+Use `labels_include_all_path` to target hosts that have all labels, `labels_include_any_path` to target hosts that have any label, or `labels_exclude_any_path` to target hosts that don't have any of the labels. Only one of `labels_include_all_path`, `labels_include_any_path`, or `labels_exclude_any_path` can be specified. If none are specified, all hosts are targeted.
```

Comment:
Maintain support for specifying an array of label names using `labels`, `labels_include_all `, `labels_include_any `, and `labels_exclude_any` for backwards compatibility.

Only one of `labels_include_all `, `labels_include_any `, `labels_exclude_any` and the new `_path` equivalents can be specified.

If more than one is specified, an error is returned and no changes are applied:
```
Error: Couldn’t edit macos_settings.custom_settings. For each configuration profile, only one of "labels_include_all_path", "labels_include_any_path", or "labels_exclude_any_path" can be included.
```